routes/car.routes.js

```diff
@@ -5,13 +5,18 @@ const router = express.Router();
 const modelController = require("../controllers/car_model.controller");
 const trimController = require("../controllers/car_trim.controller");
 const imageController = require("../controllers/car_image.controller");
+const reviewController = require("../controllers/car_review.controller");
 
 router.get("/models", modelController.listAllModels);
 
-router.get("/models/:id", modelController.findModelById);
+router.get("/model/:id", modelController.findModelById);
 
-router.get("/models/:id/trims", trimController.listTrimsById);
+router.get("/model/:id/trims", trimController.listTrimsById);
 
-router.get("/models/:id/images", imageController.listImagesById);
+router.get("/model/:id/images", imageController.listImagesById);
+
+router.get("/model/:id/reviews", reviewController.findReviewsById);
```

Comment:
I think we should have this under a different route? 
Something like `/review/model/id`? That way we can differentiate easily in the front-end

Reply:
```
router.get("/models", modelController.listAllModels);
router.get("/model/:id", modelController.findModelById);
router.get("/model/:id", trimController.listTrimsById);
router.get("/images/model/:id", imageController.listImagesById);
router.get("/reviews/model/:id", reviewController.findReviewsById);
router.post("/review/model/:id", reviewController.postReview);
```

This looks better?